src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -43,6 +45,16 @@ export default class DetailsBody extends React.Component<Props> {
       return '';
     }
 
+    const timeWindowString = `${incident.alertRule.timeWindow}m`;
+    const timeWindowInMs = intervalToMilliseconds(timeWindowString);
+    const started = moment(incident.dateStarted);
+    const end = incident.dateClosed ?? getUtcDateString(new Date());
+
+    // We want the discover chart to start at "dateStarted" - "timeWindow" - "20%"
+    const start = started
+      .subtract(timeWindowInMs, 'ms')
+      .subtract(moment(end).diff(started, 'ms') * 0.2, 'ms');
```

Comment:
this should be 20% of (end - (started - timeWindow))

Reply:
@adhiraj that could be `0` if `end - start == timeWindow`

Reply:
Updated